requirements/optional.txt

```diff
@@ -1,2 +1,2 @@
 spconv
-waymo-open-dataset-tf-2-1-0==1.2.0
+# waymo-open-dataset-tf-2-4-0
```

Comment:
Why comment this package?

Reply:
We can't find a waymo-open-dataset-tf that can install on both py3.6 and py3.9 in CI. The waymo-open-dataset-tf > 2-4-0 doesn't support py3.6, and waymo-open-dataset-tf <= 2-4-0 doesn't support py3.9.